prow/jobs/kyma/kyma-integration-k3d.yaml

```
@@ -0,0 +1,64 @@
+periodics:
+  - name: kyma-upgrade-k3d-kyma2-to-main
+    annotations:
+      description: "K3d Kyma 2.0 to main branch."
+      owner: "jellyfish"
+      pipeline.clusterprovisioning: "kyma cli"
+      pipeline.installer: "kyma deploy"
+      pipeline.platform: "k3d"
+      pipeline.test: "upgrade-fast-integration"
+      pipeline.trigger: "periodic"
+      pipeline.type: "integration"
+      testgrid-dashboards: "kyma_integration"
+    labels:
+      prow.k8s.io/pubsub.project: "sap-kyma-prow"
+      prow.k8s.io/pubsub.runID: "kyma-upgrade-k3d-kyma2-to-main"
+      prow.k8s.io/pubsub.topic: "prowjobs"
+      preset-kind-volume-mounts: "true"
+      preset-dind-enabled: "true"
+      preset-kyma-guard-bot-github-token: "true"
+    cron: "0 0 6-18/2 ? * 1-5"
+    skip_report: false
+    decorate: true
+    decoration_config:
+      grace_period: 600000000000
+      timeout: 7200000000000
+    path_alias: github.com/kyma-project/kyma
+    cluster: trusted-workload
+    extra_refs:
+      - org: kyma-project
+        repo: kyma
+        path_alias: github.com/kyma-project/kyma
+        base_ref: main
+      - org: kyma-incubator
+        repo: local-kyma
+        path_alias: github.com/kyma-incubator/local-kyma
+        base_ref: main
+    reporter_config:
+      slack:
+        channel: jellyfish-notifications
+    spec:
+      containers:
+        - image: europe-docker.pkg.dev/kyma-project/prod/testimages/e2e-dind-k3d:v20230612-dadc01a7
+          command: [ "/init.sh" ]
+          securityContext:
+            privileged: true
+            seccompProfile:
+              type: Unconfined
+            allowPrivilegeEscalation: true
+          args:
+            - "bash"
+            - "-c"
+            - |
+              chmod +x ./tests/fast-integration/prow/kyma-upgrade.sh
+              ./tests/fast-integration/prow/kyma-upgrade.sh
```

Chmod your script locally and push it to target repo. Then you don't need to chmod it in here.
```suggestion
            - ./tests/fast-integration/prow/kyma-upgrade.sh
```